src/entities/campaign/components/CampaignBanner.tsx

```
@@ -109,193 +201,195 @@ export const CampaignBanner: React.FC<CampaignBannerProps> = ({ campaignId }) =>
-          }}
-          onClick={(event) => {
-            // Prevent navigation when clicking on links
-            if (event.target instanceof HTMLAnchorElement) {
-              event.stopPropagation();
-            }
-          }}
-        />
-      </div>
+          <div
+            className="prose prose-sm max-w-none p-4"
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)